Shorten column headings

SvenRosvall opened the pull request:
Suggestions for shorter column headings to narrow the Node View table.

SvenRosvall committed:
Shorten column headings

david284 commented:
Most I agree with
I'm not sure about replacing "stored events" with "events", since it is only a count of stored events, and for many modules doesn't include the default events - maybe just 'Stored' instead?

I'm also not keen on 'Ver' instead of 'Version' - I realise the current row contents (e.g. '1a') is very short, making the title seem excessive
However, I realise it doesn't show the beta version, and if that's included (e.g. '1a B12'), then keeping the title to 'version' wouldn't seem so excessive. (I think it's need a separation between the version and beta number for clarity)
Note that it would be another message to get that parameter (for non-VLCB), but could be considered important - maybe the 'version' column is a question for Sunday?

SvenRosvall committed:
Shorten column headings

SvenRosvall commented:
Reverted Version for now. I think it would be good to show beta version. I hope the cost of one more call is acceptable.

I don't like "Stored" on its own. It doesn't say clearly what is stored. Reverted to "Stored Events". I hope we can find a better short name.
